scripts/aws/setupdb.sh

```diff
@@ -14,3 +14,14 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 
 # Run migrations
 envdir /etc/mmw.d/env /opt/app/manage.py migrate
+
+# Fetch boundary layer sql files
+FILE_HOST="http://santacruz.internal.azavea.com:9999"
+curl $FILE_HOST/boundary_huc08.sql.tar.gz | tar xz
+curl $FILE_HOST/boundary_huc10.sql.tar.gz | tar xz
+curl $FILE_HOST/boundary_huc12.sql.tar.gz | tar xz
+
+psql -f boundary_huc08.sql
```

Comment:
It's worth noting that this script leaves the `*.sql` files around, but they are happily overwritten on future runs from `tar xz`.  Uncompressed they can be several gigs, just wasn't sure if it was worth `rm`ing them.